oke.tf

```diff
@@ -42,9 +42,12 @@ module "oke" {
   services_cidr               = var.services_cidr
 
   # node pools
-  node_pools = var.node_pools
   node_pool_name_prefix = var.node_pool_name_prefix
 
+  node_pools = {

```

Comment:
I'll accept this change for now until we find why it's not taking the variables from the vars file